caffe2/core/context_gpu.h

```diff
@@ -163,6 +163,10 @@ class CUDAContext final {
   // deadlocks
   static std::mutex& mutex();
 
+  // Determine whether the device using this context can use
+  // tensor cores
+  static bool TensorCoreAvailable();
```

Comment:
That's a fair point. Originally there was some additional logic in the `CUDAContext::TensorCoreAvailable()` code, but not anymore. Where would you prefer the code to be? common_gpu or context_gpu?